src/content/docs/style-guide/writing-strategies/how-to-write-intros.mdx

```diff
@@ -0,0 +1,365 @@
+---
+title: How to write introductions
+topics:
+  - New Relic only
+  - Basic style guide
+  - Writing guidelines
+---
+Introductions are a lot like appetizers for a dinner party&mdash;they set the stage for the main course. If you put some extra work into the introduction, your readers will be ready for the main course.
+
+When you compose introductions, you typically include these three ingredients:
+
+  * Information that defines the topic
+  * Ideas that capture the interest of the user
+  * Teasers that tell the reader what to expect from the document
+
+The first two ingredients seem straightforward, but when it comes to telling readers what they can expect, we often rely on repetitive and wooden phrases. The way we handle this raises some questions: Who is doing the talking? Is it authoritative? Is it collaborative? Is this the best way to help users learn and solve their problems?
+
+We'll look at some general tips about introductions but focus on how you tell your readers what to expect from the document. If you prefer to skip the background discussion, you can jump down to the [Example lead-ins](#example-lead-ins) showing introductions that avoid stock phrases and get to the heart of the page’s subject.
+
+## Tips for more active introductions [#tips]
+
+We have a number of pages on our site that have a heading followed by a `This document is about` introductory sentence. This sentence often restates the heading. Then, we often include a paragraph later that provides a better introduction. We recommend that you avoid using the `this document is about` stock phrase because it can be repetitive and often doesn't address the reader directly.
+
+We’ll look at various alternatives to this type of construction throughout the document. Before you explore those alternatives, consider these ways to write stronger introductions:
+
+  * Focus on the user right away. Instead of starting off talking about what the document is about, talk about what the user might want to know or do (think [Five questions](/docs/style-guide/writing-strategies/five-questions-help-write-docs)).
```

Comment:
@mmfred, I'm not sure what you're suggesting here. Could you clarify it?

Reply:
@rhetoric101 - I think I was suggesting that we don't have to say "this document is about" because the page title, headings, and lead paragraph implicitly communicate this info. You can ignore if you don't feel like adding it. Or you could add this sentence just after "(think [Five questions](/docs/style-guide/writing-strategies/five-questions-help-write-docs))." 

"Let the page title, headings, and lead paragraph communicate what the document is about."